cmd/seid/cmd/iavl_parser.go

```diff
@@ -6,22 +6,34 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	dexkeeper "github.com/sei-protocol/sei-chain/x/dex/keeper"
+	dextypes "github.com/sei-protocol/sei-chain/x/dex/types"
+	minttypes "github.com/sei-protocol/sei-chain/x/mint/types"
 )
 
 type ModuleParser func([]byte) ([]string, error)
 
 var ModuleParserMap = map[string]ModuleParser{
 	"bank": BankParser,
+	"mint": MintParser,
+	"dex":  DexParser,
+}
+
+func MintParser(key []byte) ([]string, error) {
+	keyItems := []string{}
+	switch {
+	case bytes.HasPrefix(key, minttypes.MinterKey):
+		keyItems = append(keyItems, "MinterKey")
+	case bytes.HasPrefix(key, minttypes.LastTokenReleaseDate):
+		keyItems = append(keyItems, "LastTokenReleaseDate")
+	default:
+		keyItems = append(keyItems, "Unrecognized prefix")
```

Comment:
typically you would use a tombstone value for this, and it can be used across all modules (i.e. default case forDex, Bank parser as well).
basically a static string that denotes unrecognized